### ISSUE TITLE
Request body of examples lost during import

### ISSUE THREAD

**Issue by dragon-pearl** [a85/POSTMan-Chrome-Extension#10900]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When our collection is imported the examples created are missing the request raw body, even though the JSON has everything listed as expected. 

### 

### Steps To Reproduce

1. Open a new request
2. Select the body tab
3. Select the `raw` type with JSON as language
4. Enter your JSON elements
5. Run the request
6. Save the response as an example
7. Export the collection as version 2.1 or 2
8. Review the exported JSON to see the request body with expected elements
9. Import the collection
10. Open the imported request
11. Select the body tab
12. Select the `raw` type and see that the request details are empty, however the response shows as expected

### Screenshots or Videos

Request being sent:
![image](https://user-images.githubusercontent.com/54729365/167646193-b3adb71d-0a1e-45cc-a6a9-29bd9360e323.png)
Request saved as example:
![image](https://user-images.githubusercontent.com/54729365/167646734-fd854f0c-6022-43e9-be2c-c02bd980fddc.png)
Request as it appears in exported JSON
![image](https://user-images.githubusercontent.com/54729365/167647094-aadcd8c0-ec5a-4a6c-bfed-afff0e37f9d7.png)
Request example body as it appears in imported collection
![image](https://user-images.githubusercontent.com/54729365/167647931-433aed72-b0d4-4a93-b1b7-4c3bcb948ac2.png)

### Operating System

Windows/Mac

### Postman Version

9.18.1

### Postman Platform

Both

**Comment by timbochamp** [comment]
Thanks for reporting, @dragonspearl. I'm unable to reproduce this on v9.18.2 where I see my request's body of type raw and language JSON imported correctly. Are you consistently able to reproduce this? I'm wondering if you could send us some logs if so?

You can refer to these articles on how to collect the logs:

https://support.postman.com/hc/en-us/articles/360038234154-How-to-generate-a-HAR-file-
https://support.postman.com/hc/en-us/articles/360047765713-Where-can-I-find-logs-for-the-Postman-Application-

and where to send them:
https://support.postman.com/hc/en-us/requests/new

**Comment by dragon-pearl** [comment]
@timbochamp Thank you for your reply. I am able to replicate the issue every time on my account and other users'. I have submitted the logs requested through the ticket system #107189. Please let me know if there is any other assistance I can provide. 

Thank you,

**Comment by timbochamp** [comment]
Thanks! We have been able to replicate the issue and will be working on a fix. This is just happening with GET and HEAD requests though from our testing. @dragonspearl, are you facing this issue with other methods as well?

**Comment by dragon-pearl** [comment]
No, admittedly my testing has been limited to creating the collection for a company's API documentation.

**Comment by p3k** [comment]
I have the same (a similar?) issue also with POST requests. Importing the following simple test case results in a method without the examples:

```yaml
openapi: 3.0.3
info:
  title: Test API
  version: 0.0.0
  contact:
    name: Jane Doe
    email: nobody@nowhere.com
  license:
    name: Unlicensed
  description: This is just a drill.
paths:
  /foobar:
    post:
      summary: Foobar
      description: You say Hello, I say Good-bye.
      requestBody:
        content:
          text/plain:
            schema:
              type: string
            example: Hello
      responses:
        200:
          description: Success!
          content:
            text/plain:
              schema:
                type: string
              example: Good-bye
```

![image](https://github.com/postmanlabs/postman-app-support/assets/16631/de1c6986-e474-4aac-a0e4-87ccd709a47f)

**Comment by VShingala** [comment]
@p3k You can choose the value "Example" for the option named "Parameter generation" while importing under "View Import Settings" to make sure example value is used while generating collection from definition.

![Screenshot 2024-01-29 at 7 22 56 PM](https://github.com/postmanlabs/postman-app-support/assets/16756296/bb3048ca-0b4b-4d97-97cc-fbf1b2f7f1cc)

![Screenshot 2024-01-29 at 7 23 34 PM](https://github.com/postmanlabs/postman-app-support/assets/16756296/24b4c0bf-c5cc-4fbe-843a-b0ff7f56d384)
